src/components/EditEventForm/index.jsx

```diff
@@ -240,7 +241,12 @@ const EditEventForm = props => {
           <Button type="primary" htmlType="submit" className="save-button">
             Submit Edits
           </Button>
-          <Button type="danger" className="discard-button">
+          <Button
+            type="danger"
+            onClick={() => {
+              history.goBack();
```

Comment:
Should the discard button redirect you backwards or clear the form @StoneT2000? Idk here

Reply:
The button is available for existing events too so I am not sure if clearing the form would make sense for those.

Reply:
Seems good be sure to switch your base branch to nightly